DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -256,8 +260,12 @@ export class SearchSortFilterAndPaginate {
     return newResultCountMessage;
   }
 
-  private static scrollToTop(): void {
-    window.scrollTo(0, 0);
+  private scrollToTop(): void {
+    if (this.idConsideredTopForScrolling !== '') {
+      document.getElementById(this.idConsideredTopForScrolling)?.scrollIntoView();

```

Comment:
What's this new change for?

Reply:
The searchable/paginated area is below a big chunk of content, and when we interact with various controls, it scrolls to top. This allows us to define a different element at the 'top' that it scrolls to. In this case the Courses heading